packages/components/src/components/mobile-drawer/mobile-drawer.scss

```diff
@@ -114,6 +139,9 @@
             width: 16px;
             height: 16px;
             margin-right: 16px;
+            @include rtl {
```

Comment:
hi @suisin-deriv :) According to [the postcss-rtlcss docs](https://www.npmjs.com/package/postcss-rtlcss), this plugin lets us use LTR/RTL functionality without any mixins. Are you using this mixin to override RTL mirroring styles?
Another concern that I have is regarding the bundle size: As you can see on the screenshot below, this plugin creates 2 CSS classnames out of 1, which can potentially double our bundle size. Could you please check the impact on bundle size? :) 
<img width="1318" alt="Screenshot 2024-02-05 at 1 37 37 PM" src="https://github.com/binary-com/deriv-app/assets/103177211/11c466f0-f10e-4bbf-86f7-3751bb2975cc">